src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -36,6 +36,12 @@ public class ResolvePackageAssets : TaskBase
 
         public bool MarkPackageReferencesAsExternallyResolved { get; set; }
 
+        /// <summary>
+        /// Check that there is at least one package dependency in the RID graph that is not in the 
```

Comment:
This looks like an incomplete comment.  "Not in the" ...what?

Reply:
Fixed.